db/migrate/20170330091029_add_cuisine_to_resturants.rb

```diff
@@ -0,0 +1,5 @@
+class AddCuisineToResturants < ActiveRecord::Migration[5.0]
```

Comment:
In a real life scenario, we would unify adding cuisine and removing genre in one atomic migration.

We would also write a code that takes all of the existing genres and creating cuisines from them, before deleting the genre field.

No need to do it in the training project as we don't care about the persistence of the data :)